ansible/roles/common/defaults/main.yaml

```diff
@@ -6,3 +6,13 @@ bin_dir: /usr/bin
 # The directory used by Ansible to temporarily store
 # files on Ansible managed systems.
 ansible_temp_dir: /tmp/.ansible/files
+
+# The directory used for the docker config
+system_config_dir: "/etc/sysconfig"
```

Comment:
I see, it was moved by your PR [1]. Then, lines 9-11 can be removed. WDYT?

[1] https://github.com/kubernetes/contrib/pull/1015